src/Microsoft.TestPlatform.Common/Telemetry/TelemetryDataConstants.cs

```diff
@@ -8,60 +8,59 @@ namespace Microsoft.VisualStudio.TestPlatform.Common.Telemetry;
 /// <summary>
 /// The Telemetry data constants.
 /// </summary>
-[System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "CA2211:Non-constant fields should not be visible", Justification = "Part of the public API.")]
-public static class TelemetryDataConstants
+internal static class TelemetryDataConstants
 {
     // ******************** Execution ***********************
-    public static string ParallelEnabledDuringExecution = "VS.TestRun.ParallelEnabled";
+    internal static readonly string ParallelEnabledDuringExecution = "VS.TestRun.ParallelEnabled";
```

Comment:
It was probably fine to keep this level public once you changed the class to be `internal` but that's not worth changing back.

Reply:
That's a good point. I did it the other way around actually (i.e. first changed the members to `internal` and then figured I could even go for the whole class and make it `internal`), so that's why I ended up with both class and members being `internal`. I'll probably create a follow-up PR and fix this :)